src/components/Toast/index.stories.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * @Author: zhaohui
+ * @Date: 2021-05-17 14:21:43
+ * @LastEditTime: 2021-05-18 16:07:54
+ * @LastEditors: zhaohui
+ * @Description:
+ * @FilePath: /vant-react/src/components/Toast/index.stories.tsx
+ */
+import React from 'react';
+import Toast from '.';
+import Cell from '../Cell';
+import Icon from '../Icons';
+
+import '../../styles/stories.scss';
+
+export default {
+  title: 'Toast',
+  component: Toast
+};
+export const BasicUsage = () => {
+  return (
+    <div className='storybook__container column grey'>
+      <Cell
+        {...{
+          title: {
+            text: 'BaseUsage',
+            fontSize: '12px'
+          },
+          onClick: () => Toast.info({ message: 'Base Usage' })
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'Toast bottom',
+            fontSize: '12px'
+          },
+          onClick: () =>
+            Toast.info({ message: 'Toast bottom', position: 'bottom' })
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'Toast center',
+            fontSize: '12px'
+          },
+          onClick: () =>
+            Toast.info({ message: 'Toast center', position: 'center' })
+        }}
+      />
+    </div>
+  );
+};
+
+export const ToastStatus = () => {
+  return (
+    <div className='storybook__container column grey'>
+      <Cell
+        {...{
+          title: {
+            text: 'ToastSuccess',
+            fontSize: '12px'
+          },
+          onClick: () =>
+            Toast.info({ message: 'ToastSuccess', type: 'success' })
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'ToastFail',
+            fontSize: '12px'
+          },
+          onClick: () => Toast.info({ message: 'ToastFail', type: 'fail' })
+        }}
+      />
+    </div>
+  );
+};
+
+export const ToastUserSet = () => {
+  return (
+    <div className='storybook__container column grey'>
+      <Cell
+        {...{
+          title: {
+            text: 'Toast user set icon',
+            fontSize: '12px'
+          },
+          onClick: () =>
+            Toast.info({
+              message: 'Toast user set icon',
+              icon: <Icon name='good-job-o' />
+            })
+        }}
+      />
+      <Cell
+        {...{
+          title: {
+            text: 'Toast user set img',
+            fontSize: '12px'
+          },
+          onClick: () =>
+            Toast.info({
+              message: 'Toast user set img',
+              icon: <img width='28' src='https://img.yzcdn.cn/vant/logo.png' />
+            })
+        }}
+      />
+    </div>
+  );
+};
+
+export const ToastSetDefaultOptionGlobal = () => {
```

Comment:
can we also have a loading state?

Reply:
that accepts an async operation and finishes loading on resolve

Reply:
For the loading icon type, we could utilize both spinner and circular loader
![image](https://user-images.githubusercontent.com/13799221/118973161-6005c080-b93f-11eb-9982-53e180124d08.png)
![image](https://user-images.githubusercontent.com/13799221/118973183-65630b00-b93f-11eb-9cbe-916aa2d25bc0.png)

Reply:
some one code the loading coponent but don't merge......